src/Fixer/AbstractShortOperatorFixer.php

```diff
@@ -226,18 +226,15 @@ private function clearMeaningfulFromRange(Tokens $tokens, array $range): void
 
     private function isOperatorCommutative(Token $operatorToken): bool
     {
-        static $commutativeKinds = ['*', '|', '&', '^']; // note that for arrays in PHP `+` is not commutative
-        static $nonCommutativeKinds = ['-', '/', '.', '%', '+'];
-
         if ($operatorToken->isGivenKind(T_COALESCE)) {
             return false;
         }
 
-        if ($operatorToken->equalsAny($commutativeKinds)) {
+        if ($operatorToken->equalsAny(['*', '|', '&', '^'])) { // note that for arrays in PHP `+` is not commutative
```

Comment:
tried to simulate this case here for performance comparison: https://3v4l.org/3YF3b

Reply:
overall yes -> wherever we could use constants (in-place or declared), let's use constants ;) 

nonstatic -> static -> const journey over PHP versions evolution.

~static may still be needed if we actually want to determine sth in runtime, eg specific to existance of given token under given version.~ or not, we have FCT now.